### PR TITLE
Updated the Prometheus instructions

### PR DESCRIPTION
**Pull request by markpitt**
Added the two packages AspNetCore and HttpClient for dotnet add package and reduced the options to the new properties.

From what I can see this is how the package is to be used now. Hopefully, this change is useful

### CODE REVIEW

File: content/en/docs/instrumentation/net/exporters.md [hunk 1]
```diff
@@ -205,7 +205,8 @@ docker run \
 Next, install the Prometheus exporter:
 
 ```
-dotnet add package OpenTelemetry.Exporter.Prometheus
+dotnet add package OpenTelemetry.Exporter.Prometheus.AspNetCore --prerelease
+dotnet add package OpenTelemetry.Exporter.Prometheus.HttpListener --prerelease
```
Comment (reyang) [inline content/en/docs/instrumentation/net/exporters.md:209]: Why is this needed?

Comment (cijothomas) [reply content/en/docs/instrumentation/net/exporters.md:209]: This doc seems to be attempting to show both asp.net core app and non asp.net core app. If that is the intention, then need both packages. I think its best to stick with asp.net core here,. (the non asp.net core one is anyway not prodcution recommended)

@markpitt 
In the aspnetcore part, the doc is missing important step : https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Exporter.Prometheus.AspNetCore#step-3-configure-prometheus-scraping-endpoint

Comment (markpitt) [reply content/en/docs/instrumentation/net/exporters.md]: @cijothomas Thanks for the link. I'd been looking for some more detail on how it's configured. 

I've added the the registration of the middleware and removed the HttpClient package. Hopefully, this fixes all the issues

**Comment by svrnm** [comment]
Please run `npm run prettier:write` to fix the styling issues.

**Review by reyang — APPROVED** [review]

**Review by cartermp — APPROVED** [review]
